pkg/apis/core/types.go

```diff
@@ -5686,6 +5712,17 @@ type TopologySpreadConstraint struct {
 	// This is an alpha field and requires enabling MinDomainsInPodTopologySpread feature gate.
 	// +optional
 	MinDomains *int32
+	// NodeInclusionPolicy instructs how we treat nodeAffinity/nodeTaint policy
+	// when calculating pod topology spread skew.
+	// - NodeAffinity(nodeAffinity/nodeSelector):
+	//   - Respect (default): nodes matching nodeAffinity/nodeSelector will be included.
+	//   - Ignore: all nodes will be included.
+	// - NodeTaint(node taints):
+	//   - Respect: nodes without taints, along with tainted nodes that tolerate the incoming pod, will be included.
+	//   - Ignore (default): all nodes will be included.
+	// This is a alpha-level feature enabled by the PodTopologySpreadNodePolicy feature flag.
+	// +optional
+	NodeInclusionPolicy *NodeInclusionPolicy
```

Comment:
In several places, like TopologyKey, we define "eligible domain" as nodes that match the node affinity. This is no longer true. Those docs should cross reference the NodeInclusionPolicy.

Reply:
done :)